packages/round-manager/src/features/program/ViewProgramPage.tsx

```diff
@@ -56,9 +54,7 @@ export default function ViewProgram() {
   const [hasAccess, setHasAccess] = useState(true);
   const [grantType, setGrantType] = useState<
     "quadraticFunding" | "directGrant" | undefined
-  >(
-    process.env.REACT_APP_DIRECT_GRANT_ENABLED ? undefined : "quadraticFunding"
-  );
+  >();
```

Comment:
If there should be no default, explicitly setting this as `undefined` would make that clear (though `null` would be a better choice for "intentionally empty").

Reply:
brb drafting an ecma proposal for `Option<T>` in js..

on a more serious note, i'll change this to null